com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -776,6 +787,14 @@ public ShaderVariantLogLevel shaderVariantLogLevel
             set { m_ShaderVariantLogLevel = value; }
         }
 
+        /// <summary>
+        /// Returns the selected update mode for volumes.
+        /// </summary>
+        public VolumeFrameworkUpdateMode volumeFrameworkUpdateMode
+        {
+            get { return m_VolumeFrameworkUpdateMode; }
+        }
```

Comment:
you can simplify this : `public VolumeFrameworkUpdateMode volumeFrameworkUpdateMode  => m_VolumeFrameworkUpdateMode;`

Reply:
Done.